azure-mgmt-datalake-store/tests/test_mgmt_datalake_store.py

```diff
@@ -21,25 +24,116 @@ def setUp(self):
             azure.mgmt.datalake.store.DataLakeStoreAccountManagementClient
         )
 
+    @ResourceGroupPreparer(location=_REGION)
+    def test_vnet_operations(self, location):
+        account_name = self.get_resource_name('adlsacct')
+        vnet_rule_name = self.get_resource_name('vnetrule1')
+        resource_group = 'lewu-rg'
+        subnet_id = '/subscriptions/9e1f0ab2-2f85-49de-9677-9da6f829b914/resourceGroups/lewu-rg/providers/Microsoft.Network/virtualNetworks/lewuVNET/subnets/default'
+        subscription_id = '9e1f0ab2-2f85-49de-9677-9da6f829b914'
+
+        authority_host_uri = 'https://login.microsoftonline.com'
+        tenant = '72f988bf-86f1-41af-91ab-2d7cd011db47'
+        authority_uri = authority_host_uri + '/' + tenant
+        RESOURCE = 'https://management.core.windows.net/'
+        client_id = '565a758f-4919-40b8-af45-765cbb13664a'
+        client_secret = 'ycCDVcjcBuOwwroCdtlo4ZNzMoJ3iQ70JQExHmL8LnA='
```

Comment:
You can't do that in the test, you just made public this credentials and you're good to kill this SP now :(
Please read the tutorial for credentials:
https://github.com/Azure/azure-sdk-for-python/wiki/Contributing-to-the-tests